src/unity/lib/image_util.cpp

```diff
@@ -230,13 +235,41 @@ std::vector<std::string> get_directory_files(std::string url, bool recursive) {
   return ret;
 }
 
+
+bool has_ending (std::string fullString, std::string ending) {
```

Comment:
To scope this function to just this `.cpp` file (internal linkage), mark it as `static`.